### PR TITLE
AzureCP PR #1 for May Release

### PR DESCRIPTION
**Pull request by huangpf**
The content to be added to Changelog is as follows:
* Update Tests
* AVSet Command
* Premium VM & VMSS Commands

### CODE REVIEW

**Comment by huangpf** [comment]
@amarzavery @haocs Could you please help review? Thanks.

File: test/commands/arm/vm/arm.availset-tests.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ describe('arm', function() {
         });
       });
 
-      it('list-available-sizes should list the available VM sizes in availability set', function (done) {
+      it.skip('list-available-sizes should list the available VM sizes in availability set', function (done) {
```
Comment (amarzavery) [inline test/commands/arm/vm/arm.availset-tests.js]: why is the test skipped?

File: package.json [hunk 2]
```diff
@@ -126,6 +126,7 @@
     "unit-arm-vm": "node scripts/unit.js testlist-arm-vm.txt",
```
Comment (amarzavery) [inline package.json]: @huangpf - I just merged a PR in node sdk for asm compute.Do you need to publish that package and consume it here?

File: lib/commands/arm/vmss/vmss._js [hunk 3]
```diff
@@ -122,7 +122,7 @@ function createStorageResources(storageManagementClient, resourceGroupName, name
       var stoParams = {};
       stoParams.name = generateNewStorageAccountName(name + 'sto' + i.toString());
       stoParams.location = location;
-      stoParams.accountType = 'Standard_GRS';
+      stoParams.accountType = storageType ? storageType : 'Premium_LRS';
```
Comment (haocs) [inline lib/commands/arm/vmss/vmss._js]: Is the default type changed from 'Standard_GRS' to 'Premium_LRS'?

File: package.json (hunk 2 above)
Comment (huangpf) [inline package.json]: Yes, please~ Thank you~

File: lib/commands/arm/vmss/vmss._js (hunk 3 above)
Comment (huangpf) [inline lib/commands/arm/vmss/vmss._js]: Yes, it's called the 'Premium VM' feature, :-)

@sauryadas

Comment (yaxia) [inline lib/commands/arm/vmss/vmss._js]: Just a note that when we update to azure-arm-storage 0.13.0-preview, it should be: 
stoParams.sku = { name: storageType ? storageType : 'Premium_LRS' }

**Comment by yaxia** [comment]
I don't know why the CI shows a green light while there is actual failure in it.

    arm webapp create should work:   ․ arm webapp create should work: 185ms
    arm webapp list should work: 
    error: missing required argument `resource-group'
    1) arm webapp list should work  
    End:3:32:50 AM

@haocs, I think it is caused by your [changed ](https://github.com/Azure/azure-xplat-cli/commit/753c149b5f7f5bd1e6548a8f28dd89fe09218d2b) which makes <resource-group> mandatory but no test code is updated.

**Comment by huangpf** [comment]
Thanks @yaxia .

Yes, there was a test failure unrelated to my code, and I had to revert the test. I think somewhere it enforced your test run.

**Comment by huangpf** [comment]
@haocs

File: lib/commands/arm/vmss/vmss._js (hunk 3 above)
Comment (huangpf) [inline lib/commands/arm/vmss/vmss._js]: When is it going to update to this major version?

Comment (yaxia) [inline lib/commands/arm/vmss/vmss._js]: I am running tests at my side. If everything goes well, I will send PR next Monday.

**Comment by haocs** [comment]
@huangpf @yaxia let me fix it.

File: lib/commands/arm/vmss/vmss._js (hunk 3 above)
Comment (huangpf) [inline lib/commands/arm/vmss/vmss._js]: @amarzavery If this is the case, can our PR go first as for now?

Comment (amarzavery) [inline lib/commands/arm/vmss/vmss._js]: @huangpf - I am fine with it. Make sure that you consume the latest storage package and make changes accordingly.

**Pull request merged**